### ISSUE TITLE
Host Report A Goose somewhere faster

### ISSUE THREAD

**Issue by illicitonion** [CodeYourFuture/curriculum#948]
https://github.com/CodeYourFuture/Project-Record-A-Goose hosts itself on Render (https://record-a-goose.onrender.com/) which has a super-slow spin-up time. I just went to the page and it took >40 seconds for first load.

We should move this over to... Whatever our supported express hosting platform is these days.

/cc @sztupy I can't remember where we settled on hosting platforms?

**Comment by sztupy** [comment]
I haven't gotten around to finish the Vercel guide, so currently the proposal is Netlify + Supabase. This app definitely needs an upgrade so it can work with either of them though.

Is this a project that we wish to keep? If yes I can take a look to move it over, wouldn't mind the exercise.

**Comment by illicitonion** [comment]
Yeah, this is used in ITP (the "Record a Goose" exercise in https://programming.codeyourfuture.io/user-data/sprints/2/backlog/) - if you wouldn't mind picking it up that'd be great! I imagine it'd make for a nice sample PR for others to follow in the future!

**Comment by SallyMcGrath** [comment]
It's a good workshop that brings a lot of value, so would good to rescue

**Comment by sztupy** [comment]
Cool. It doesn't have a database (uses an in-memory session store with a fallback to cookie based one) so moving it to our Netlify account shouldn't be too hard.
